tap_hubspot/__init__.py

```diff
@@ -96,6 +96,8 @@ class StateFields:
     "forms":                "/forms/v2/forms",
     "workflows":            "/automation/v3/workflows",
     "owners":               "/owners/v2/owners",
+
+    "tickets":              "/crm/v4/objects/tickets",
```

Comment:
Is this `v3` or `v4` ? it is mentioned as `v3` in the provided docs

> To request a list of all tickets, make a GET request to /crm/v3/objects/tickets.

Reply:
It is `v4`, if you check the API doc, the latest version of the CRM APIS, it will be `v4` 
Here, the requirement is to list the tickets with their associations and for this, I have used the endpoint `/crm/v4/objects/tickets/{ticketId}/associations/{toObjectType} ` mentioned in the endpoint section of [API doc link](https://developers.hubspot.com/docs/api/crm/tickets)